src/logger/CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+cmake_minimum_required(VERSION 3.16)
+
+set(LOGGER_SOURCES Logger.cpp)
+
+add_library(kp_logger ${LOGGER_SOURCES})
```

Comment:
Would be good to get more context on the thinking for this logging module, including context in relation with the python side as it wouldn't really be used in the same level, ideally the logging structure is not very opinionated in the core library itself, it's also referenced in other sections about making spdlog an optional deps as well as discussed in more detail in the logging files comments